blocks/bricks/sequence_generators.py

```diff
@@ -282,6 +308,8 @@ def get_dim(self, name):
             return self.transition.get_dim(name)
         elif name == 'outputs':
             return self.readout.get_dim(name)
+        elif self.language_model and name in self.language_model.apply_step.outputs:
```

Comment:
Can you please add a shortcut `self._lm_state_names` for `self.language_model.apply_step.outputs`? This would look uniform together  with `self._state_names`, `self._glimpse_names`, etc.